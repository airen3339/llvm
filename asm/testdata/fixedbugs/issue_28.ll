declare void @g()

define void @f() {
	call void () @g()
	ret void

	call void () @g()
	ret void
}
