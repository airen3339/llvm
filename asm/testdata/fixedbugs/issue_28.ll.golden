declare void @g()

define void @f() {
; <label>:0
	call void @g()
	ret void
; <label>:1
	call void @g()
	ret void
}
